core/src/Revolution/modElement.php

```diff
@@ -275,6 +275,13 @@ public function getTag()
                     if (is_scalar($value)) {
                         $propTemp[$key] = $key . '=`' . $value . '`';
                     } else {
+                        if(is_array($value)){
+                            foreach($value as $v){
+                                if(is_object($v)){
+                                    $value = '';
+                                }
+                            }
+                        }
                         $propTemp[$key] = $key . '=`' . (is_array($value) ? md5(serialize($value)) : md5(uniqid(rand(), true))) . '`';
```

Comment:
Wouldn't it be better to wrap this line with `try/catch` and use the `md5(uniqid(rand(), true)))` in the catch block?

Reply:
Please check my last commit, I test it and look like no 500 error appear